test/unit/tasks/interfaces_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'test_helper'
+require 'rake'
+
+class InterfacesTest < ActiveSupport::TestCase
+  let(:null_store) { ActiveSupport::Cache.lookup_store(:null_store) }
```

Comment:
I am looking into Rails cache in test environment and I wonder - what is purpose of this null store hack? We seem to clean cache before and after each test.